qa/L0_implicit_state/implicit_state.py

```diff
@@ -83,6 +83,80 @@ def test_wrong_implicit_state_name(self):
         err_str = str(e.exception).lower()
         self.assertIn("state 'undefined_state' is not a valid state name", err_str)
 
+    def test_implicit_state_single_buffer(self):
```

Comment:
Is is easy enough to test that growable memory isn't used (or returns error) when set to false?  ex: set growable_memory to false, send request that tries to grow state tensor, assert failure

Reply:
You can grow the buffer even when you don't use `growable_memory`. The only catch is that the old memory is thrown away and a new memory is returned to the user.

Reply:
I see, thanks